ios/PurchasesHybridCommon/PurchasesHybridCommon.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,16 +1,14 @@
 {
-  "object": {
-    "pins": [
-      {
-        "package": "swift-snapshot-testing",
-        "repositoryURL": "https://github.com/pointfreeco/swift-snapshot-testing",
-        "state": {
-          "branch": null,
-          "revision": "f29e2014f6230cf7d5138fc899da51c7f513d467",
-          "version": "1.10.0"
-        }
+  "pins" : [
+    {
+      "identity" : "swift-snapshot-testing",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/pointfreeco/swift-snapshot-testing",
+      "state" : {
+        "revision" : "f29e2014f6230cf7d5138fc899da51c7f513d467",
+        "version" : "1.10.0"
       }
-    ]
-  },
-  "version": 1
+    }
+  ],
+  "version" : 2
```

Comment:
This breaks compatibility with Xcode 13 ⚠️  you'll need to revert this file.

Reply:
Yup just did 👍